test/unit/test.inversify.config.ts

```diff
@@ -35,6 +37,9 @@ const logger: Winston.Logger = Winston.createLogger({
 container.bind<MessageFactory>(TYPES.MessageFactory).to(MessageFactory).inSingletonScope();
 container.bind<MessageHandlerFactory>(TYPES.MessageHandlerFactory).to(MessageHandlerFactory).inSingletonScope();
 
+const mockPool = stubInterface<IConnectionPool>();
+container.bind<IConnectionPool>(TYPES.ConnectionPool).toConstantValue(mockPool);
```

Comment:
@croutledgeaais In general, this technique could be used in the individual tests without going through inversify -- Does it provide enough value to do in this way so that all tests get the same mock object by default?

Also, there are some Dummy classes implemented in the code to satisfy an interface that likely could be replaced with stubInterface<IInterfaceTy>(). Thoughts?